apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/TraceIgnoreExtendService.java

```diff
@@ -76,4 +85,10 @@ public boolean trySampling(final String operationName) {
     public void forceSampled() {
         super.forceSampled();
     }
+
+    void handleTraceIgnorePatternsChanged() {
+        if (StringUtil.isNotBlank(traceIgnorePatternWatcher.getTraceIgnorePathPatterns())) {
```

Comment:
What happens if I set the value as blank? I think the PR misses some logic here.

Reply:
Blank string is considered as `DELETE` event, and it will use the default value (`TraceIgnorePatternWatcher#51`) of `IgnoreConfig.Trace.IGNORE_PATH`, so `traceIgnorePatternWatcher.getTraceIgnorePathPatterns()` may be effectively not blank unless `IgnoreConfig.Trace.IGNORE_PATH` is set to empty